docs/decisions/0001-purpose-of-this-repo.rst

```diff
@@ -6,52 +6,42 @@ Status
 
 **Draft**
 
-.. TODO: When ready, update the status from Draft to Provisional or Accepted.
-
-.. Standard statuses
-    - **Draft** if the decision is newly proposed and in active discussion
-    - **Provisional** if the decision is still preliminary and in experimental phase
-    - **Accepted** *(date)* once it is agreed upon
-    - **Superseded** *(date)* with a reference to its replacement if a later ADR changes or reverses the decision
-
-    If an ADR has Draft status and the PR is under review, you can either use the intended final status (e.g. Provisional, Accepted, etc.), or you can clarify both the current and intended status using something like the following: "Draft (=> Provisional)". Either of these options is especially useful if the merged status is not intended to be Accepted.
-
 Context
 *******
 
-TODO: Add context of what led to the creation of this repo.
+With the `deprecation of ecommerce`_, Axim launched a funded project to develop capabilities of connecting Open edX to external ecommerce system. The intention is to connect Open edX with a third-party e-commerce platform, enabling a straightforward implementation process and making the e-commerce service accessible for various initiatives. The request for proposal can be found here: `tCRIL Funded Contribution - WooCommerce Discovery`_.
 
-.. This section describes the forces at play, including technological, political, social, and project local. These forces are probably in tension, and should be called out as such. The language in this section is value-neutral. It is simply describing facts.
+For this integration project, we create a `WooCommerce Discovery document`_, which proposes the creation of a WordPress plugin and an API to generate the necessary actions on the Open edX side. The following image illustrates the expected interactions.
 
-Decision
-********
+.. image:: ../_static/enrollment-diagram.png
+  :alt: Sequence diagram of purchase using Woocommerce Integration.
```

Comment:
Is there a plan to document other flows, say unenrollment or refunds and also sad path flows like 500 on attempted enrollment?

Reply:
The scope of this document is only the purpose of this repo. I use that image to give context, but if it creates confusion about the scope of the API, I prefer to remove it.

About other documentation, I expect to have in this API decision around the API. I think the correct place to put the flow decisions may be in the [WooCommerce Plugin](https://github.com/eduNEXT/openedx-woocommerce-plugin) because that plugin will be the flow's handler. What do you think?

Reply:
I would not remove it, maybe just say it's included to show the high level flow and not meant to be a comprehensive list of use cases and user flows.